lua/cmp/autocmds.lua

```diff
@@ -0,0 +1,130 @@
+local M = {}
+
+local v = vim.version()
+local has_native_autocmd = v.major >= 0 and v.minor >= 7
```

Comment:
I think this is going to be removed because we don't need fallback, but this can go wrong if neovim version becomes v1.0 (v.major == 1 and v.minor == 0)